plugins/woocommerce/client/admin/client/settings-payments/components/payment-gateway-list-item/payment-gateway-list-item.tsx

```diff
@@ -32,83 +34,100 @@ export const PaymentGatewayListItem = ( {
 			return 'needs_setup';
 		}
 		if ( gateway.state?.enabled ) {
-			if ( isWCPay ) {
+			if ( isWcPay ) {
 				if ( gateway.state?.test_mode ) {
 					return 'test_mode';
 				}
 			}
 			return 'active';
 		}
 
+		if ( isWcPay ) {
+			return 'recommended';
+		}
+
 		return 'inactive';
 	};
 
-	return {
-		key: gateway.id,
-		className: `transitions-disabled woocommerce-item__payment-gateway ${
-			isWCPay ?? `woocommerce-item__woocommerce-payment`
-		} ${ hasIncentive ?? `has-incentive` }`,
-		title: (
-			<>
-				{ gateway.title }
-				{ hasIncentive ? (
-					<StatusBadge
-						status="has_incentive"
-						message={ __(
-							'Save 10% on processing fees',
-							'woocommerce'
+	return (
+		<div
+			id={ gateway.id }
+			className={ `transitions-disabled woocommerce-list__item woocommerce-list__item-enter-done woocommerce-item__payment-gateway ${
+				isWcPay ?? `woocommerce-item__woocommerce-payment`
+			} ${ hasIncentive ?? `has-incentive` }` }
+			{ ...props }
+		>
+			<div className="woocommerce-list__item-inner">
+				<div className="woocommerce-list__item-before">
+					<DefaultDragHandle />
+					<img src={ gateway.icon } alt={ gateway.title + ' logo' } />
+				</div>
+				<div className="woocommerce-list__item-text">
+					<span className="woocommerce-list__item-title">
+						{ gateway.title }
+						{ hasIncentive ? (
+							<StatusBadge
+								status="has_incentive"
+								message={ __(
+									'Save 10% on processing fees',
```

Comment:
We need to use the dynamic badge part of the incentive data.

Reply:
I'll address this in https://github.com/woocommerce/woocommerce/pull/53432. 🙂

Reply:
Oh! Yeah. Mixed up PRs :)